bagOfWords.py

```diff
@@ -0,0 +1,61 @@
+import re
+import csv
+from bs4 import BeautifulSoup
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from sklearn.feature_extraction.text import CountVectorizer
```

Comment:
```suggestion
import csv
import re

from bs4 import BeautifulSoup
from nltk.corpus import stopwords
from nltk.tokenize import word_tokenize
from sklearn.feature_extraction.text import CountVectorizer
```

For our Python style, we work with PEP-8, so your code would probably be refactored as this. It's nothing serious, just to let you know how we work.

Reply:
I will keep that in mind while I code.